### PR TITLE
Get rid of wildcard imports

### PR DESCRIPTION
**Pull request by cmichi**
Follow-up to #1886.

### CODE REVIEW

File: core/client/src/client.rs [hunk 1]
```diff
@@ -61,7 +61,7 @@ use crate::in_mem;
 use crate::block_builder::{self, api::BlockBuilder as BlockBuilderAPI};
 use crate::genesis;
 use consensus;
-use substrate_telemetry::*;
+use substrate_telemetry::{SUBSTRATE_INFO};
```
Comment (bkchr) [inline core/client/src/client.rs]:
```suggestion
use substrate_telemetry::SUBSTRATE_INFO;
```

File: core/service/src/lib.rs [hunk 2]
```diff
@@ -41,7 +44,7 @@ use exit_future::Signal;
 pub use tokio::runtime::TaskExecutor;
 use substrate_executor::NativeExecutor;
 use parity_codec::{Encode, Decode};
-use tel::*;
+use tel::{SUBSTRATE_INFO};
```
Comment (bkchr) [inline core/service/src/lib.rs]:
```suggestion
use tel::SUBSTRATE_INFO;
```

File: core/cli/src/lib.rs [hunk 3]
```diff
@@ -19,6 +19,9 @@
 #![warn(missing_docs)]
 #![warn(unused_extern_crates)]
 
+#[macro_use(telemetry)]
+extern crate substrate_telemetry;
```
Comment (thiolliere) [inline core/cli/src/lib.rs]: I think in 2018 we should import macro by namespace: use substrate_telemetry::telemetry;

File: core/client/src/lib.rs [hunk 4]
```diff
@@ -20,6 +20,9 @@
 #![warn(missing_docs)]
 #![recursion_limit="128"]
 
+#[macro_use(telemetry)]
+extern crate substrate_telemetry;
```
Comment (thiolliere) [inline core/client/src/lib.rs]: same

File: core/consensus/aura/src/lib.rs [hunk 5]
```diff
@@ -26,6 +26,9 @@
 //! Blocks from future steps will be either deferred or rejected depending on how
 //! far in the future they are.
 
+#[macro_use(telemetry)]
+extern crate substrate_telemetry;
```
Comment (thiolliere) [inline core/consensus/aura/src/lib.rs]: same

File: core/finality-grandpa/src/lib.rs [hunk 6]
```diff
@@ -52,6 +52,9 @@
 //! or prune any signaled changes based on whether the signaling block is
 //! included in the newly-finalized chain.
 
+#[macro_use(telemetry)]
+extern crate substrate_telemetry;
```
Comment (thiolliere) [inline core/finality-grandpa/src/lib.rs]: same

File: core/service/src/lib.rs [hunk 7]
```diff
@@ -19,6 +19,9 @@
 
 #![warn(unused_extern_crates)]
 
+#[macro_use(telemetry)]
+extern crate tel;
```
Comment (thiolliere) [inline core/service/src/lib.rs]: same

File: core/cli/src/lib.rs (hunk 3 above)
Comment (bkchr) [reply core/cli/src/lib.rs]: If it is one macro, that could be done per import. If you need multiple macros like `info`, `error` etc, I would still vote for `macro_use`.

Comment (thiolliere) [reply core/cli/src/lib.rs]: I don't have strong opinion on it actually but I think we had converted every macro_use to namespace import, for example for decl_* macros : https://github.com/paritytech/substrate/blob/master/srml/example/src/lib.rs#L23

**Pull request merged**